apache-trusted-release/evaluate.md

```diff
@@ -17,12 +17,12 @@ In this phase the ATR will check claims about the release artifacts to enforce p
 3. Dependencies are acceptably licensed.
 4. Release artifacts have correct GPG detached signatures and checksums.
 5. Reproducible build claims are validated.
-6. SBOMs are well formed and have proper claims.
+6. SBOMs are well-formed and have proper claims.
```

Comment:
Plenty of language resources say that hyphens are dying out in English. I don't know if there is empirical evidence to back this up, but I avoid hyphens in preparation for when the puck is potentially moving.